code/build.gradle.kts

```diff
@@ -19,7 +19,7 @@ buildscript {
         mavenLocal()
     }
     dependencies {
-        classpath("com.github.adobe:aepsdk-commons:gp-3.0.2")
+        classpath("com.github.adobe:aepsdk-commons:gp-3.4.0")
```

Comment:
Can you use `3.4.1` that we release yesterday. This has some edge-case fixes to the plugin.